### PR TITLE
[doxy] add missing docs and adjust docs in GUIInfoManager

### PR DESCRIPTION
**Pull request by xhaggi**
see title

### CODE REVIEW

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -4243,393 +4248,393 @@ const infomap playlist[] =       {{ "length",           PLAYLIST_LENGTH },
 /// @{
 /// \table_start
 ///   \table_h3{ Labels, Type, Description }
-///   \table_row3{   <b>`Pvr.IsRecording`</b>,
-///                  \anchor Pvr_IsRecording
+///   \table_row3{   <b>`PVR.IsRecording`</b>,
+///                  \anchor  IsRecording
```
Comment (ksooo) [inline xbmc/GUIInfoManager.cpp]: Why not `PVR_IsRecoring`? We use a prefix everywhere else.

Comment (xhaggi) [reply xbmc/GUIInfoManager.cpp]: Accidently removed, thx

**Comment by xhaggi** [comment]
@ronie any objections?

File: xbmc/GUIInfoManager.cpp [hunk 2]
```diff
@@ -4026,7 +4034,7 @@ const infomap visualisation[] =  {{ "locked",           VISUALISATION_LOCKED },
 ///   \table_row3{   <b>`Fanart.Image`</b>,
 ///                  \anchor Fanart_Image
 ///                  _string_,
-///     todo
+///     Returns the fanart image for the parent TV Show
```
Comment (ronie) [inline xbmc/GUIInfoManager.cpp]: not limited to TV Shows only, works for Albums as well.

Comment (xhaggi) [reply xbmc/GUIInfoManager.cpp]: All of that is c/p from wiki

Comment (ronie) [reply xbmc/GUIInfoManager.cpp]: i already mentioned that on slack when someone referred to the inline comments as "the truth".

it's not. what we have in core is a 1:1 c/p of community contributed interpretations of what those infolabels mean.  bugs, errors, incompleteness and all included.

one may wonder why we copied that info to our core in the first place...

Comment (xhaggi) [reply xbmc/GUIInfoManager.cpp]: I'm with you, will change it to what you suggested

**Comment by xhaggi** [comment]
@ronie addressed your comments.

**Comment by ronie** [comment]
great, should be good to go then.

File: xbmc/GUIInfoManager.cpp [hunk 3]
```diff
@@ -1609,17 +1609,17 @@ const infomap musicpartymode[] = {{ "enabled",           MUSICPM_ENABLED },
...
+///     Mood of the currently playing song
 ///   }
 ///   \table_row3{   <b>`MusicPlayer.PlaylistPlaying`</b>,
 ///                  \anchor MusicPlayer_PlaylistPlaying
 ///                  _boolean_,
-///     Todo
+///     Returns true if a playlist is currently playing
 ///   }
-///   \table_row3{   <b>`MusicPlayer.Exists`</b>,
+///   \table_row3{   <b>`MusicPlayer.Exists(relative,position)`</b>,
```
[the remark this answers is not in the document]
Comment (MartijnKaijser) [reply xbmc/GUIInfoManager.cpp]: was this solved in pushed commit so we can merge this?

[the remark this answers is not in the document]
Comment (xhaggi) [reply xbmc/GUIInfoManager.cpp]: Yep it was.

**Comment by xhaggi** [comment]
@MartijnKaijser meh I want to fixup my commit history before merge.

**Comment by MartijnKaijser** [comment]
sorry, missed that one :(

**Comment by xhaggi** [comment]
np .. next time 😄

**Pull request merged**